### PR TITLE
RAS: Update My Account button breakpoints, spacing

### PR DESCRIPTION
**Pull request by laurelfulford**
### All Submissions:

* [x] Have you followed the [Newspack Contributing guideline](https://github.com/Automattic/newspack-plugin/blob/master/.github/CONTRIBUTING.md)?
* [x] Does your code follow the [WordPress' coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/) and [VIP Go coding standards](https://vip.wordpress.com/documentation/vip-go/code-review-blockers-warnings-notices/)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?

### Changes proposed in this Pull Request:

To be tested with https://github.com/Automattic/newspack-theme/pull/2051 -- more information and testing steps are available in that issue!

### Other information:

* [x] Have you added an explanation of what your changes do and why you'd like us to include them?
* [ ] Have you written new tests for your changes, as applicable?
* [ ] Have you successfully ran tests with your changes locally?

### CODE REVIEW

File: assets/reader-activation/auth.scss [hunk 1]
```diff
@@ -20,21 +20,21 @@
...
 
-			@media screen and ( min-width: 960px ) {
+			@media screen and ( min-width: 1168px ) {
 				display: none;
 			}
 
 			a {
 				display: flex;
-				padding: 0.4em;
+				padding: 7px;
```
Comment (dkoo) [inline assets/reader-activation/auth.scss:29]: Should we use `rem` instead, in case the base font size changes?

**Review by dkoo — APPROVED** [review]
Looks good—I left a question but I'm not sure if it needs to be addressed. Will leave that decision up to you.

**Comment by laurelfulford** [comment]
Thanks @dkoo! 

I'm going hold off merging this one until the related theme PR is ready; I'm also going to take a closer look at your suggestion as I work on that one!

**Comment by adekbadek** [comment]
Hi @laurelfulford – it's been a while since the last update here :D What's the status of the mentioned theme PR?

**Comment by laurelfulford** [comment]
The theme PR that this one went with is both super-stale, and also changes too much -- I think touching the logo size is just asking for issues. I've going to close it and this one, and start fresh with something lower-impact overall.

**Pull request closed without merging**